telegram/poll.py

```diff
@@ -48,6 +48,42 @@ def de_json(cls, data, bot):
         return cls(**data)
 
 
+class PollAnswer(TelegramObject):
+    """
+    This object represents an answer of a user in a non-anonymous poll.
+
+    Attributes:
+        poll_id (:obj:`str`): Unique poll identifier.
+        user (:class:`telegram.User`): The user, who changed the answer to the poll.
+        option_ids (List[:obj:`int`]): Identifiers of answer options, chosen by the user.
+
+    Args:
+        poll_id (:obj:`str`): Unique poll identifier.
+        user (:class:`telegram.User`): The user, who changed the answer to the poll.
+        option_ids (List[:obj:`int`]): 0-based identifiers of answer options, chosen by the user.
+            May be empty if the user retracted their vote.
+
+    """
+    def __init__(self, poll_id, user, option_ids, **kwargs):
+        self.poll_id = poll_id
+        self.user = user
+        self.option_ids = option_ids
+
```

Comment:
I'd set `_id_attrs` but probably that's rather for #1724

Reply:
well, it was there for poll. But hmm, I would keep it in your other PR as well.